CHANGELOG.md

```
@@ -1,3 +1,74 @@
+- 
+  [``] by 
+- 
+  [`* https://github.com/piqueterron/OtherMediator/pull/11#issuecomment-3199403821`] by 
+- 
+  [``] by 
+- 
+  [`The following files were modified:`] by 
+- 
+  [``] by 
+- 
+  [`* `src/OtherMediator.Contracts/Unit.cs``] by 
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)